src/views/privacypolicy/privacypolicy.jsx

```diff
@@ -203,6 +203,7 @@ var Privacypolicy = React.createClass({
                              Policy on a regular basis.
                         </p>
                     </section>
+                    <p><b>The Scratch Privacy Policy was last updated: October 2016</b></p>
```

Comment:
@kaschm is this supposed to be emboldened text in the page too, or was it just bold in the issue you created to make it distinct?